101-azurerelay-create-namespace/azuredeploy.json

```diff
@@ -0,0 +1,36 @@
+{
+    "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "name": {
+            "type": "string",
+            "metadata": {
+                "description": "Name of the Azure Relay namespace"
+            }
+        }
+    },
+    "variables": {
+        "location": "[resourceGroup().location]",
+        "apiVersion": "2016-07-01",
+        "defaultSASKeyName": "RootManageSharedAccessKey",
+        "defaultAuthRuleResourceId": "[resourceId('Microsoft.Relay/namespaces/authorizationRules', parameters('name'), variables('defaultSASKeyName'))]"
+    },
+    "resources": [{
+        "apiVersion": "[variables('apiVersion')]",
```

Comment:
please use a raw string instead of a variable (here and in all other apiVersions)